DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -20,6 +20,8 @@
 from DQMOffline.L1Trigger.L1TriggerDqmOffline_cff import *
 from DQM.CTPPS.totemDQM_cff import *
 
+hcalOfflineSourceSequence.remove(tpTask)
```

Comment:
I doubt HCAL will be too happy to see their sequence being modified.
Get someone from HCAL to clarify - also this should be done via hcalOfflineSourceSequence, not hack-removed via DQMOffline_cff.py